docker/gen-run-image

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+ROOT=$(git rev-parse --show-toplevel)
+
+export DIST="xenial"
+export DMD="dmd1"
+export F="production"
+
+cd $ROOT
+BEAVER_DOCKER_Iimg="sociomantictsunami/dhtnode-builder" beaver dlang install
+beaver dlang make pkg
+
+rm -rf docker-tmp
+mkdir docker-tmp
+cp build/$F/pkg/dhtnode-d*.deb docker-tmp/dhtnode.deb
```

Comment:
It doesn't seem like `dhtnode-common` package is needed because docker installation won't use systemctl/upstart anyway.

Reply:
Ha, finally my package separation pays off (?)